data-prepper-plugins/opensearch/README.md

```diff
@@ -138,6 +138,10 @@ If not provided, failed records will be written into the default data-prepper lo
 all the records received from the upstream prepper at a time will be sent as a single bulk request.
 If a single record turns out to be larger than the set bulk size, it will be sent as a bulk request of a single document.
 
+- `document_id_field` (optional): A string of document identifier which is used as `id` for the document when it is stored in the OpenSearch. Each incoming record is searched for this field and if it is present, it is used as the id for the document, if it is not present, a unique id is generated by the OpenSearch when storing the document. If the field has "/" in it then the incoming record is searched in the json sub-objects instead of just in the root of the json object. For example, if the field is specified as `info/id`, then the root of the event is searched for `info` and if it is found, then `id` is searched inside it. The value specified for `id` is used as the document id
```

Comment:
Thank you for adding this missing documentation!

The documentation should clarify that this uses the standard Data Prepper JSON Pointer syntax as is used elsewhere for retrieving the value.